docs/components.md

```diff
@@ -55,6 +55,18 @@ Independent, small visual UI elements to use in applications.
 * [service-telemetry](https://github.com/mozilla-mobile/android-components/tree/master/components/service/telemetry) - A generic library for sending telemetry pings from Android applications to Mozilla's telemetry service.
 * [service-glean](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean) - A generic library for sending telemetry pings from Android applications to Mozilla's telemetry service (eventually replacing `service-telemetry`).
 
+## Samples
+
+* [Browser (simple)](https://github.com/mozilla-mobile/android-components/blob/master/samples/browser) - very basic browser implementation based on the browser components.
```

Comment:
I'm not sure what the best title for this link would be so that it's consistent with my Reference Browser suggestion below. Maybe "Sample Browser (simple browser)" or "Browser (simple browser)"?

Reply:
The goal for keeping this name was:
a) Put simple/reference browser atop since they are samples for wide range of features.
b) Keep list sorted in alphabetical order.

So I'll use second suggestion (and will apply it to reference browser as well). What do you think?